vfio_user/src/lib.rs

```diff
@@ -298,7 +298,7 @@ impl Client {
             major: 0,
             minor: 1,
         };
-        info!("Command: {:?}", version);
+        debug!("Command: {:?}", version);
```

Comment:
I'll let @rbradford decide here since he introduced the code with the `info` log level, and I don't want to go back and forth regarding this topic.

Reply:
There is docs/logging.md that tries to give rationale for different levels of logging. But since this PR and also #3382 wants to downgrade it to `debug!()` then I guess it is too spammy for `info!()` so you can go ahead and downgrade it.